build/Build.cs

```diff
@@ -110,7 +110,9 @@ ITargetDefinition TestDefinition(ITargetDefinition targetDefinition, Target depe
                     .SetConfiguration(Configuration)
                     .SetFramework(framework)
                     .EnableNoBuild()
-                    .EnableNoRestore());
+                    .EnableNoRestore()
+                    .EnableBlameCrash()
+                    .SetBlameCrashDumpType("full"));
```

Comment:
full, since mini would not contain the heap which is needed to view async stacks afaik